<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mw_rotate</class>
 <widget class="QWidget" name="mw_rotate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rotate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="hs_rotate">
       <property name="minimum">
        <number>-360</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_rotate">
       <property name="minimum">
        <number>-360</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_expand">
       <property name="text">
        <string>expand</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Resample</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="rb_nearest">
         <property name="text">
          <string>Nearest</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bG_resample</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_bicubic">
         <property name="text">
          <string>Bicubic</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bG_resample</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_bilinear">
         <property name="text">
          <string>Bilinear</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bG_resample</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="ch_rotation_center">
       <property name="text">
        <string>Rotation center:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="x_label">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_x"/>
     </item>
     <item>
      <widget class="QLabel" name="y_label">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_y"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_apply_rotation">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bG_resample"/>
 </buttongroups>
</ui>
